fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -29,6 +29,7 @@ def self.upload_build(upload_url, ipa, options)
 
         begin
           connection.post do |req|
+            req.options.timeout = options.delete(:timeout)
```

Comment:
Instead of doing `options.delete(:timeout)`, can we just pass `timeout` as a parameter into this method?